src/WebJobs.Script/FileAugmentation/PowerShell/profile.ps1

```diff
@@ -0,0 +1,20 @@
+﻿# Azure Functions profile.ps1
+#
+# This profile.ps1 will get executed every "cold start" of your Function App.
```

Comment:
This is not accurate, as this will will run with `JobHost` restarts and specialization, correct?

Reply:
Yes, that's true in the host, this is the literal provided by Eamon, I'll get that corrected.